plutus-core/plutus-core/src/PlutusCore/DeBruijn.hs

```diff
@@ -12,155 +12,117 @@ module PlutusCore.DeBruijn
     , NamedTyDeBruijn (..)
     , FreeVariableError (..)
     , AsFreeVariableError (..)
-    , deBruijnTy
-    , deBruijnTerm
-    , deBruijnProgram
-    , unDeBruijnTy
-    , unDeBruijnTerm
-    , unDeBruijnProgram
-    , unNameDeBruijn
-    , unNameTyDeBruijn
-    , fakeNameDeBruijn
+    , runDeBruijnT
```

Comment:
I had to change them to reflect the naming in UPLC. I don't particularly like my new naming